examples/webgl_gpu_picking_point.html

```diff
@@ -0,0 +1,283 @@
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+	<style>
+		body {
+			color: #cccccc;
+			font-family: Monospace;
+			font-size: 13px;
+			text-align: center;
+
+			background-color: #050505;
+			margin: 0px;
+			overflow: hidden;
+		}
+
+		#info {
+			position: absolute;
+			top: 0px;
+			width: 100%;
+			padding: 5px;
+		}
+
+		a {
+			color: #0080ff;
+		}
+
+	</style>
+</head>
+<body>
+
+<div id="container"></div>
+<div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - gpu - picking - point
+</div>
+
+<script type="module">
+/* 
+	Why use gpu to pick the object point?
+	1.Picking in cesium using gpu depth to calculate picking points，I think threejs can also have.
+	2.It is more efficient to calculate the picking point using gpu depth when the number of faces of a single model is large.
+	3.I can release the geometry data under the mesh, free more memory, and not affect the rendering.
+*/
+import * as THREE from '../build/three.module.js';
+import Stats from './jsm/libs/stats.module.js';
+import { GUI } from './jsm/libs/dat.gui.module.js';
+import { OrbitControls } from './jsm/controls/OrbitControls.js';
+
+
+var renderCamera, scene, renderer, stats, pickingScene, gui;
+
+var mesh;
+var mesh1;
+
+var controls;
+var raycaster;
+
+var pickingCamera;
+var pickingTexture;	
+
+var GPUPickingPoint = new THREE.Vector3();
+var CPUPickingPoint = new THREE.Vector3();
+
+init();
+animate();
+
+function init() {
+
+	//
+
+	raycaster = new THREE.Raycaster();
+	renderCamera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 10000 );
+	pickingCamera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 1, 10000 );
+	renderCamera.position.x = 200;
+
+	scene = new THREE.Scene();
+	pickingScene = new THREE.Scene();
+
+	//
+
+	var axesHelper = new THREE.AxesHelper( 100 );
+	scene.add( axesHelper );
+
+	//
+
+	// var geometry = new THREE.SphereBufferGeometry(100, 20, 20);
+	var geometry = new THREE.BoxBufferGeometry( 100, 100, 100 );
+
+	var material = new THREE.MeshNormalMaterial();
+	var depthMaterial = new THREE.MeshDepthMaterial();
+	depthMaterial.depthPacking = THREE.RGBADepthPacking;
+
+	mesh = new THREE.Mesh( geometry, material );
+	mesh1 = new THREE.Mesh( geometry, depthMaterial );
+	scene.add( mesh );
+	pickingScene.add( mesh1 );
+
+	//
+
+	renderer = new THREE.WebGLRenderer( { antialias: true, logarithmicDepthBuffer: true } );
+	renderer.setPixelRatio( window.devicePixelRatio );
+	renderer.setSize( window.innerWidth, window.innerHeight );
+	document.body.appendChild( renderer.domElement );
+
+	//
+	
+	pickingTexture = new THREE.WebGLRenderTarget(renderer.domElement.width, renderer.domElement.height);
+
+	//
+	
+	stats = new Stats();
+	document.body.appendChild( stats.dom );
+
+	//
+
+	gui = new GUI();
+	gui.add( renderer.capabilities, 'logarithmicDepthBuffer' );
```

Comment:
What is the point of this menu item?

Reply:
@Mugen87 @mrdoob  `renderer.capabilities` probably should not be exposed like this.

Reply:
I want know  that setting does useful for MeshDepthMaterial, maybe i should check the shader. if that's work, the way of compute the picking point will be different.

Reply:
> renderer.capabilities probably should not be exposed like this.

I think it's okay like that. The `capabilities` object can be used to check for `WebGL2` (`renderer.capabilities.isWebGL2`) or get the maximum anisotropy (`renderer.capabilities.getMaxAnisotropy()`). I prefer this style than adding more methods/flags to `WebGLRenderer`.

Reply:
@Mugen87 @mrdoob The OP is trying to _set_ the capability.

Reply:
@1147079942 Please perform no write operations on `renderer.capabilities`. It's best when you remove the respective line.

Reply:
Well, a typical user error...

Reply:
@Mugen87 I'll resolve my own questions.

Reply:
i event don't know you guys have understand what i express

Reply:
let's try again, take this as a question.In my opinio, `logarithmicDepthBuffer` is work in shader, this use a different algorithm to calculate the value between near and far.I think this `logarithmicDepthBuffer` option design should work for `MeshDepthMaterial` too. If is the same in the threejs like what i say, or threejs change the `MeshDepthMaterial` shader, enable the `logarithmicDepthBuffer` in the `MeshDepthMaterial`, this example will be broken.

Reply:
You can only enable or disable logarithmic depth buffer in the `WebGLRenderer` constructor.

See, for example, https://threejs.org/examples/webgl_camera_logarithmicdepthbuffer.html